website/frontend/management/commands/scraper.py

```diff
@@ -244,7 +269,7 @@ def add_to_git_repo(data, filename, article):
     else:
         already_exists = True
 
-
+    # When close this file?
```

Comment:
Good catch!  This is relying on CPython's refcounts immediately garbage collecting.  Would be good to switch.